pillar.example

```diff
@@ -34,7 +38,7 @@ postgres:
   # POSTGRES
   # Append the lines under this item to your postgresql.conf file.
   # Pay attention to indent exactly with 4 spaces for all lines.
-  postgresconf: |
+  postgresconf: |-
```

Comment:
Otherwise an unnecessary blank line rendered, more noticeable if both `listen_addresses` and `port` are given.